wgpu-hal/src/gles/adapter.rs

```diff
@@ -15,7 +15,7 @@ impl super::Adapter {
     /// resulting in an `Err`.
     /// # Notes
     /// `WebGL 2` version returned as `OpenGL ES 3.0`
-    fn parse_version(mut src: &str) -> Result<(u8, u8), crate::InstanceError> {
+    fn parse_version(mut src: &str) -> Result<((u8, u8), bool), crate::InstanceError> {
```

Comment:
Instead of doing all of this rigamarole with returning a bool, can't we just do `cfg!(target_arch = "wasm32")` for webgl?

Reply:
Yeah, that seems fine.